pkg/cloudvolume/login.go

```diff
@@ -34,17 +34,23 @@ func newLoginCommand() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVar(&opts.host, "host", "", "Cloud Volumes portal hostname/ip")
-	cmd.Flags().StringVarP(&opts.username, "username", "u", "", "cloudvolume username")
-	cmd.Flags().StringVarP(&opts.password, "password", "p", "", "cloudvolume passowrd")
+	cmd.Flags().StringVar(&opts.host, "host", "", "HPE Cloud Volumes portal hostname/ip")
```

Comment:
The empty quotes is where you can specify a default value, so you can put the default host value there.